### DISCUSSION TITLE
Flowise Integration Input Schema

### DISCUSSION THREAD

**Discussion by JLMoon24** [ItzCrazyKns/Vane#93] [Q&A] [answered]
I'm trying to use Perplexica in FlowiseAi.  When I try to create a tool, its asking for an input schema (Property, type, description).  There are 2 different JSON's available which are the package and the tsconfig.  Any help would be greatly appreciated.  Thanks for reading!

**Comment by ItzCrazyKns** [comment, marked as answer]
> I'm trying to use Perplexica in FlowiseAi. When I try to create a tool, its asking for an input schema (Property, type, description). There are 2 different JSON's available which are the package and the tsconfig. Any help would be greatly appreciated. Thanks for reading!

The package file in the root directory is for the backend and the backend code lies in the `src` folder. Inside the UI folder the package file is for the frontend.

**Comment by JLMoon24** [comment]
Thanks for the hasty reply.
